gufe/components/proteincomponent.py

```diff
@@ -53,6 +54,11 @@
 class ProteinComponent(ExplicitMoleculeComponent):
     """Wrapper around a Protein representation.
 
+    Relative to a SmallMoleculeComponent, this representation additionally contains information
+    relating to the residue and chain information.  This is achievable by having the MonomerInfo attributes
+    present on each atom of the input RDKit molecule, which is done when reading from either PDB or `.mae`
```

Comment:
Is mae support sorted out now?

Reply:
mae is in rdkit already, and it populates the MonomerInfo thing, so it should be fine yes.  Probably should do a quick how-to on the openfe docs pages